SpatialGDK/Source/SpatialGDK/Public/Interop/ActorSystem.h

```diff
@@ -188,6 +191,9 @@ class ActorSystem
 
 	TMap<Worker_EntityId_Key, TSet<Worker_ComponentId>> PendingDynamicSubobjectComponents;
 
+	// Stores subobjects from bnetloadonclient actors that have gone out of the client's interest
+	TMap<Worker_EntityId_Key, TMap<FUnrealObjectRef, FNetworkGUID>> EntityRemovedDynamicSubObjects;
+
```

Comment:
```suggestion
```
This is not needed in the ActorSystem now, right?

Reply:
true whoops, should've caught that